README.md

```diff
@@ -74,9 +75,17 @@ This is what the state machine looks like:
 LinK configuration is entirely done by setting environment variables.
 
 - `INTERFACE`: Name of the interface where LinK should add and remove IPs.
+- `HOSTNAME`: Name of the host.
 - `USER`: Username used for basic auth
 - `PASSWORD`: Password used for basic auth
 - `PORT` (default: 1313): Port where the LinK HTTP interface will be available
+- `KEEPALIVE_INTERVAL`: Duration of the lease given to an VIP. If a node is down, it can take up to KEEPALIVE_INTERVAL seconds to failover.
```

Comment:
```suggestion
- `KEEPALIVE_INTERVAL`: Duration of the lease given to a VIP. If a node is down, it can take up to KEEPALIVE_INTERVAL seconds to failover.
```